.env

```diff
@@ -1,2 +1,2 @@
 VUE_APP_API_CLIENT="server"
-VUE_APP_BACKEND_URL="http://localhost:1337"
\ No newline at end of file
+VUE_APP_BACKEND_URL="https://mitra-social.herokuapp.com"
```

Comment:
Don't think we want to change this ;) maybe we should add a `.env.dist` file with the defaults and don't check in the `.env` file?